### ISSUE TITLE
Add sort entries option to Browse feature

### ISSUE THREAD

**Issue by typhainepl** [ProteinsWebTeam/interpro7-client#313]
It would be nice to have an option to sort InterPro and member databases entries ordered ascending or descending. 
So that we can see the latest created entries first.

**Comment by gustavo-salazar** [comment]
The API currently doesn't support entries sorting by date. 
I think this will be possible to do for entries, but I'd be worry that the follow up of it, would be some sorting capabilities for other endpoints. Which is not technically possible at the moment. We have that data in elasticsearch and when we display it, it is using aggregations and sorting those is really expensive, basically not possible. 
I guess the question is are we happy to provide this feature, given the lack of consistency it will create in the browse pages.

**Comment by gustavo-salazar** [comment]
@matthiasblum What's your opinion on this one?

**Comment by matthiasblum** [comment]
I agree that it would be nice. I'm sometimes a bit frustrated not being able to sort results by columns, like on [UniProt](https://www.uniprot.org/uniprot/?query=*&sort=id&desc=yes). 

However, I don't think we had a single request from users asking this feature, so I believe it is not that important. Also I agree with Gustavo, adding this feature in one endpoint only adds confusion and inconsistency. 

Finally, should someone really wants to sort results, they can fetch data from the REST API and sort them on their side.

TL;DR: would be nice if it was easy/straightforward to implement but it isn't so I am fine without.

**Comment by gustavo-salazar** [comment]
Cool, so. I'm going to close this story, and if users request this, we can reconsider and research if there is a way to do it with decent performance
